### ISSUE TITLE
Depth texture support - let's collect the games (and GE dumps if available)

### ISSUE THREAD

**Issue by hrydgard** [hrydgard/ppsspp#13256]
Some games use depth buffers as textures, generally through CLUT16 or CLUT32, so they can look up the depth value in a palette to perform various effects like custom fog, depth of field etc. 

Usually they access the depth buffer through address 0x042xxxxx - that 2 tells the memory controller to swizzle around the data from the depth swizzle to the texture swizzle - so these accesses are actually quite quick to detect.

I plan to add support for this, so let's collect the games that do this - ideally by linking to an issue that has GE frame dumps.

Me and my Katamari: #6411  (fog effect)

(probably) Silent Hill Origins/Shattered Memories: #6265 (very tricky shadow mapping)

Gensou Suikoden: #5205 (depth of field) (no GE frame dump)

Kurohyo 2: #8390 (depth of field) (no GE frame dump)

Silent Hill Zero? #8497

**Comment by Panderner** [comment]
How about Ford Bold Moves Street Racing Fog Effect? (#12294)

**Comment by hrydgard** [comment]
I am not sure about that one - it could be one, haven't investigated yet. But thanks for the reminder.

**Comment by DonelBueno** [comment]
I don't think this is related, but maybe it is: https://github.com/hrydgard/ppsspp/issues/10229

**Comment by hrydgard** [comment]
Not likely, but it's also a depth problem indeed.

**Comment by nassau-tk** [comment]
Can I do an amateur question?

Briefly speaking, if something in which drawing before and behind the object isn't right is found, should it be reported?

**Comment by hrydgard** [comment]
@nassau-tk If it renders wrong, it should be in an issue. Whether it belongs in this one or not is hard to say though - show some pictures.

**Comment by nassau-tk** [comment]
@hrydgard 

Thank you for teach for me!
Please,see attached screenshot and dump.
[UCJS10094_0001.zip](https://github.com/hrydgard/ppsspp/files/5213980/UCJS10094_0001.zip)

**[Minna no Sukkiri (Eng:Everybody Stress Buster)]**
![everybodystress_background](https://user-images.githubusercontent.com/48179091/93016582-52a07580-f5fd-11ea-913e-12af73be79b5.png)

**[Maybe,These type dialog are drawing with blur processing on background.]**
These dialog are "Autosave warnig".
![everybodystress_autosavedialog01](https://user-images.githubusercontent.com/48179091/93016776-9fd11700-f5fe-11ea-9890-6f93c175987f.png)

![everybodystress_autosavedialog02](https://user-images.githubusercontent.com/48179091/93016779-a2cc0780-f5fe-11ea-94c4-84e7cb49f629.png)

Is this a not relation?

**Comment by nassau-tk** [comment]
@hrydgard 

And, #6979 is not related?

**Comment by hrydgard** [comment]
I'll check the dumps later. It's possible that it's related though would be a little bit surprising if they used depth texturing for simple menu effects. Could also be some other kind of bug.

**Comment by nassau-tk** [comment]
Related #12626

**Comment by Panderner** [comment]
> * NEW: Ford Bold Moves #12994

@hrydgard It's #12294 not #12994

**Comment by Panderner** [comment]
Current status of Silent Hill games as we mentioned in #6265 and #8497:

Shattered Memories:

OpenGL:
![Screenshot_2021-01-03-14-53-39-864_org ppsspp ppsspp](https://user-images.githubusercontent.com/37503397/103473487-31de7480-4dd4-11eb-9148-c7e5562285ca.jpg)

Vulkan:
![Screenshot_2021-01-03-14-53-53-568_org ppsspp ppsspp](https://user-images.githubusercontent.com/37503397/103473490-34d96500-4dd4-11eb-9092-4c1b92499bb4.jpg)

GE Dump: 
[ULUS10450_0001.zip](https://github.com/hrydgard/ppsspp/files/5761302/ULUS10450_0001.zip)

Origins:

OpenGL:
![Screenshot_2021-01-03-14-44-12-021_org ppsspp ppsspp](https://user-images.githubusercontent.com/37503397/103473523-8124a500-4dd4-11eb-9d10-16ceddf00bb1.jpg)

Vulkan:
![Screenshot_2021-01-03-14-44-23-211_org ppsspp ppsspp](https://user-images.githubusercontent.com/37503397/103473528-926db180-4dd4-11eb-8711-bd2f25e67039.jpg)

GE dump: 
[ULUS10285_0001.zip](https://github.com/hrydgard/ppsspp/files/5761303/ULUS10285_0001.zip)

**Comment by nassau-tk** [comment]
Got a [Depth / Z] label #13950 .

**Comment by hrydgard** [comment]
Yes, but it's not a depth texture issue.
